malariagen_data/anoph/dipclust.py

```diff
@@ -634,16 +663,16 @@ def plot_diplotype_clustering_advanced(
 
         fig["layout"]["yaxis"]["title"] = "Distance (manhattan)"
 
-        if transcript and amino_acids and not df_snps.empty:
+        if df_aa is not None:
```

Comment:
Small change here to ensure that variables are always bound. Previously it was possible that the `df_snps` variable might be unbound.